packages/nimble-components/src/table-column/base/index.ts

```diff
@@ -156,6 +151,13 @@ export abstract class TableColumn<
         this.setAttribute('slot', this.internalUniqueId);
     }
 
+    /** @internal */
+    public cellViewTagChanged(): void {
```

Comment:
Pretty sure you can make this `protected` or even `private`.

Reply:
Updated to `protected`.